taskApp/src/app/pages/list-tasks/list-tasks.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="centered-content">
+    <div class="row mt-1" style="height:80px;"> 
```

Comment:
Just as a heads up, we try to avoid inline styling on our clients.  This just keeps the code clean especially when there is a lot of css (which there usually is because we write our own).